app/lib/submission_builder/ty2024/states/nj/documents/schedule_nj_hcc.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module SubmissionBuilder
+  module Ty2024
+    module States
+      module Nj
+        module Documents
+          class ScheduleNjHcc < SubmissionBuilder::Document
+            include SubmissionBuilder::FormattingMethods
+
+            def schema_file
+              SchemaFileLoader.load_file("us_states", "unpacked", "NJIndividual2024V0.1", "NJCommon", "SchNJHCC.xsd")
+            end
+
+            def document
+
+              build_xml_doc("SchNJHCC") do |xml|
+                xml.HealthCovAllYear 'X'
+              end
+            end
+
+            private
+            
+            def intake
+              @submission.data_source
+            end
+            
+            def calculated_fields
+              @nj2450_fields ||= intake.tax_calculator.calculate
+            end
```

Comment:
[pebble] -  we'd want to rename the `nj2450_fields`, but actually we don't need this `calculated_fields` yet on this since the `X` is always there, so removing this and `intake` might be better (unless required by a protocol/superclass or something?)

Reply:
good catch - copy paste error!